macros/schema_tests/equal_sum.sql

```diff
@@ -0,0 +1,34 @@
+{% macro test_equal_sum(model) %}
+
+{% set column_name = kwargs.get('column_name', kwargs.get('arg')) %}
+{% set compare_model = kwargs.get('compare_model', kwargs.get('arg')) %}
+{% set compare_column_name = kwargs.get('compare_column', kwargs.get('arg')) %}
+
+{#-- Prevent querying of db in parsing mode. This works because this macro does not create any new refs. #}
```

Comment:
You can remove this part -- it's only useful if the macro tries to execute a query (like getting the columns in a table) during parsing.